convene-web/app/views/rooms/show.html.erb

```diff
@@ -1,3 +1,11 @@
+<% content_for :stylesheets do %>
+  .top-menu { display: none }
+<% end %>
+
+<% content_for :head do %>
+  <title><%= "#{current_workspace.name} - #{current_room.name}" %></title>
```

Comment:
We may want an application helper for `page_title`, which we can either override on a per-controller basis or let stand.